### DISCUSSION TITLE
How can make text lines have less space?

### DISCUSSION THREAD

**Discussion by gerroon** [TriliumNext/Trilium#2090] [Q&A]
Hi

The line spacing between the text lines in the Text format is way too much for my taste. How can I make the spaces between the text line less, like almost half of what it is now.

thanks

**Comment by abitofevrything** [comment]
By pressing shift+enter you will create a line break instead of a paragraph, removing all extra spacing created by pressing enter.

Protip: this generally works in any paragraph-based editor, such as Google Docs or Microsoft Word, so this is not just limited to Trilium.

**Comment by gerroon** [comment]
Thanks but is there a way to just condense the overall spacing, some kind of theme/css trick that I am not myself equipped to make it.

Also using Shift Enter is not useful when using indenting, it indents all the lines below with the line break thing. So I would like t a theme visual solution :(

**Reply by zadam** [reply]
You can create a custom CSS (CSS code note with label `#appCss`) with something like:

```css
.note-detail-editable-text p {
    margin: 0; /* or 3px or whatever */
}
```

and restart

**Comment by gerroon** [comment]
Thanks, @zadam that seems to be pretty close to what I need.
